OpenSeadragonHTMLelements.js

```diff
@@ -88,12 +89,21 @@
       if (e !== null) {
         const vpRect = this.viewer.viewport.imageToViewportRectangle(e.rect)
         const vpPos = viewer.viewport.imageToViewportCoordinates(e.rect.x, e.rect.y)
-        this.viewer.viewport.fitBoundsWithConstraints(new OpenSeadragon.Rect(
-          vpPos.x - vpRect.width / 2,
-          vpPos.y - vpRect.height / 2,
-          vpRect.width,
-          vpRect.height
-        ))
+        if ("animation" in e && e.animation == 'pan'){
+          this.viewer.viewport.panTo(new OpenSeadragon.Point(
```

Comment:
In my testing, your suggestion pans to the element origin. If that's the behaviour you need, then its fine, but changing should make it pan to the center of the element.

```suggestion
          this.viewer.viewport.panTo(new OpenSeadragon.Point(vpPos.x, vpPos.y), false)
```

Reply:
Thanks, I'll make this change. I hadn't really noticed on my project because the overlays are pretty small, but it does make sense to center the center of the overlay.